tests/bsim/bluetooth/mesh/src/test_beacon.c

```diff
@@ -315,6 +315,8 @@ static struct {
 	uint32_t iv_index;
 #if CONFIG_BT_MESH_V1d1
 	uint8_t random[13];
+	uint64_t pp_hash;
```

Comment:
What does `pp` mean?

Reply:
Private proxy

Reply:
Usually `pp` means pointer to pointer in many coding styles.